### PR TITLE
MONA-147 Style the default Loading Indicator.

### PR DESCRIPTION
**Pull request by acsearles**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -124,7 +129,56 @@
               </table>
             </div>
           </div>
-    
+          
+          <div class="col-md-12" style="padding-top:28px">
+            <accordion close-others="oneAtATime">
+              <accordion-group heading="This is an Open Accordion." is-open="true">
+                <div class="cg-busy-wrapper" cg-busy="{promise:indicator.promise,templateUrl:indicator.templateUrl,message:indicator.message,backdrop:indicator.backdrop,delay:indicator.delay,minDuration:indicator.minDuration}">
```
Comment (mstone6769) [inline index.html]: .cg-busy-wrapper class? Shouldn't the directive do this?

File: lib/src/iui-loading-indicator.scss [hunk 2]
```diff
@@ -1,4 +1,111 @@
...
+  min-height: 200px;
+}
+
+.cg-busy {
+  position: absolute;
+  top: 0px;
+  left: 0px;
+  right: 0px;
+  bottom: 0px;
+  z-index: 1001;
```
Comment (mstone6769) [inline lib/src/iui-loading-indicator.scss]: I have a feeling we may run into issues with this. Can you make it a variable?

File: index.html (hunk 1 above)
Comment (mstone6769) [inline index.html]: It's possible to use an attribute CSS selector to target any element with an attribute of cg-busy.

Comment (acsearles) [inline index.html]: I'll try that.

File: lib/src/iui-loading-indicator.scss (hunk 2 above)
Comment (acsearles) [inline lib/src/iui-loading-indicator.scss]: Sure. I'll even look into why we need it. It's a carry over from their styles.

File: lib/dist/iui-module.js [hunk 3]
```diff
@@ -10,45 +10,35 @@
...
+      link: function link(scope, element, attrs) {
+
+        var currentMessage = document.getElementById('loading-button').innerHTML;
+        var loadingMessage = 'Demoing';
+
+        console.log(attrs.iuiLoadingLabel);
+        console.log(element);
+
+        if (attrs.loading) {
+          console.log(loadingMessage);
```
Comment (mstone6769) [inline lib/dist/iui-module.js]: Console?

Comment (acsearles) [inline lib/dist/iui-module.js]: oh yeah this is stuff that's still in the directive that I'm trying to build for the loading label. This shouldn't do anything for the cg-busy stuff.

I should have put it in a branch but I didn't realize that until I had merged it all into the code base.

Comment (acsearles) [inline lib/dist/iui-module.js]: I can go back there and pull all that out of there and put it in another directive if you'd like.

**Pull request merged**